.github/workflows/_lint.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         # It doesn't matter how you change it, any change will cause a cache-bust.
         working-directory: ${{ inputs.working-directory }}
         run: |
-          poetry install --with lint,typing
```

Comment:
ooc why is this needed? we want to lint without the testing dependencies since they might mask an invalid import